Assets/Scripts/NarrativeManager.cs

```diff
@@ -0,0 +1,47 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class NarrativeManager : MonoBehaviour {
+    public  GameObject    textArea;
+    public  NarrationItem startingNarrativeItem;
+    private NarrationItem currentNarrativeItem;
+    public AudioSource   audioSource;
+    private Coroutine     audioCoroutine;
+
+    private void Start() {
+        currentNarrativeItem = startingNarrativeItem;
+        RunNarrativeItem();
+    }
+
+    public void AdvanceNarrative(NarrationItem next, int option = 0) {
+        StopPreviousItem();
+        currentNarrativeItem = next;
+        RunNarrativeItem();
```

Comment:
Since you have made this, there is no use to the function that I have hooked up to the button previously, you think we should delete the old one and continue with this one?

Reply:
Again, I didn't see functionality like this in your code. So I started working on this system.

Reply:
you do you.